src/styles/_global-nav.scss

```diff
@@ -202,13 +202,88 @@ $btn-selected-blue: #004A95;
   }
 }
 
+.global-nav--language-picker {
+  @media (max-width: $screen-md) {
+    padding: 0;
+  }
+  @media (min-width: $screen-md) {
+    &:hover {
+      color: $btn-hover-green;
+    }
+  }
+
+  display: flex;
+  align-items: center;
+  color: $black;
+  background-color: transparent;
+  background: none;
+  border: none;
+  padding: 8px 0 0 0;
+}
+
+.lang-selector-wrapper {
+  position: relative;
+}
+
+.lang-picker-mobile {
+  display: none;
+  @media (max-width: $screen-md) {
+    display: block;
+    margin-left: 16px;
+    font-size: 16px;
+    font-family: inherit;
+  }
+}
+
+@media (max-width: $screen-md) {
+  .globe-icon {
+    padding-top: 6px;
+  }
+}
+
+.lang-picker {
+  background: #FFFFFF;
+  display: none;
+  position: absolute;
+  box-shadow: 0px 0px 8px 0px #1616161F;
+  z-index: 30;
+  margin-top: 5px;
+  left: -21px;
+}
+
+.lang-dropdown {
+  width: 160px;
+}
+
+.lang-option {
+  display: block;
+  color: $black;
+  margin: 4px 0px;
+  padding: 4px 20px;
+  font-size: 15px;
+  font-family: "Roboto", "Helvetica Neue", Helvetica, Arial, sans-serif;
+  line-height: 18px;
+  &:hover {
+    color: $btn-hover-blue;
+    font-weight: 700;
+    transition: all 200ms;
+    transition-timing-function: cubic-bezier(.4,0,.2,1);
+    transition-duration: .15s;
+  }
+}
+.lang-active {
+  display: block;
+}
+
```

Comment:
Let's leverage sass nesting so that we can scope our values to a single top level class here:
```scss
.global-nav--language-picker {
  @media (max-width: $screen-md) {
    padding: 0;
  }
  @media (min-width: $screen-md) {
    &:hover {
      color: $btn-hover-green;
    }
  }

  display: flex;
  align-items: center;
  color: $black;
  background-color: transparent;
  background: none;
  border: none;
  padding: 8px 0 0 0;

  .lang-selector-wrapper {
    position: relative;
  }

  .lang-picker-mobile {
    display: none;
    @media (max-width: $screen-md) {
      display: block;
      margin-left: 16px;
      font-size: 16px;
      font-family: inherit;
    }
  }

  @media (max-width: $screen-md) {
    .globe-icon {
      padding-top: 6px;
    }
  }

  .lang-picker {
  // scss code
  }

  .lang-dropdown {
   // etc  
  }

  .lang-option {
   // etc
  }
  .lang-active {
    // etc
  }
}
```

If we do this, then you can actually remove the name-spacing of css classes - right now `lang-picker` and `lang-dropdown` are global classes that can be accessed and applied to any html element in the repo and it will apply its styling. If we "scope" it to a "top-level-class" we can be sure that the styles will only apply to html elements within the parent class (`global-nav--language-picker` in this case)

Reply:
Great call, Im not as familiar with sass nesting so id love to go over this with you later!